packages/integrations/useAxios/index.ts

```diff
@@ -176,9 +176,15 @@ export function useAxios<T = any, R = AxiosResponse<T>, D = any>(...args: any[])
     waitUntilFinished().then(onFulfilled, onRejected)
   const execute: OverallUseAxiosReturn<T, R, D>['execute'] = (executeUrl: string | AxiosRequestConfig<D> | undefined = url, config: AxiosRequestConfig<D> = {}) => {
     error.value = undefined
-    const _url = typeof executeUrl === 'string'
+    let _url = typeof executeUrl === 'string'
       ? executeUrl
-      : url ?? ''
+      : url ?? config.url
+
+    if (_url === undefined) {
+      error.value = new AxiosError(AxiosError.ERR_INVALID_URL)
+      _url = ''
```

Comment:
Are we sure we want to reassign the url? If we do then the axios request will still fire even though we gave the user an error, right?

Reply:
Yeah, you are right. Updated the code. Thanks for reviewing! 💚